whynotearth/BrowTricks
# 341 add phone number verification

# Issue Being Addressed

#341 

# Type of PR

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[x] Update to Existing Feature
[ ] Other (state below)

# Description

**For New Features:** 
- Add verify page for phone numbers
- Handle server errors better

** For Updates **
Improved signup/login fields (keyboard hints)

# How to Test/Reproduce

Signup then verify. Also check error handling for both client side errors and server errrors.

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

If the verification code starts with `0`, you probably see an error, refresh page. that's a temporary api error.  #352 

⚠ Other features with input text fields _may_ be affected because of updates to error handling.
